libminifi/include/core/ProcessContext.h

```diff
@@ -90,6 +91,7 @@ class ProcessContext : public controller::ControllerServiceLookup, public core::
     return processor_node_;
   }
 
+  // TODO(fgerlits): remove if possible
```

Comment:
Having TODOs is fine, but either they need fixing before merge, or the description needs to be a bit more detailed: why consider removing these, when, why it's not yet possible, etc.
It may be someone else who eventually gets around to doing the TODO. I'm also fine with not having the TODO, if it doesn't add too much value, just wanted to explain my thought process a bit better than 2 days ago. Not sure if that's the case here.

Reply:
I added these TODOs quite early on, when I still thought I could get rid of the `Property` and `Relationship` classes completely.  We might still do that eventually, but I don't think it will be soon, and it will require changes in many places, not just these three.  So removing the TODOs is probably the best option.